src/pages/Home/components/Card Container/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+.buttons_container {
+    max-width: 70vw;
+    display: flex;
+    margin: 5em auto;
+    justify-content: space-between;
+    gap: 20vw;
+}
+
+@media only screen and (max-width: calc(300px + 12em + 10em)) {
```

Comment:
If this media is being used for mobile or tablet etc. Create separate file for sass mixins and define them there. For example $mobile-breakpoint: 490px; and u can import them and use in media queries